Fakebook.Profile/Fakebook.Profile.DataAccess/Services/ProfileService.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+using Fakebook.Profile.DataAccess.BlobModel;
+using MongoDB.Bson;
+
+namespace Fakebook.Profile.DataAccess.Services
+{
+    /// <summary>
+    /// This class is to 
+    /// </summary>
+    public class ProfileService : IProfileService
+    {
+        /// <summary>
+        /// Get a Specirfic Profile
+        /// </summary>
+        /// <param name="email"></param>
+        /// <returns></returns>
+#pragma warning disable CS1998 // Async method lacks 'await' operators and will run synchronously
+        public async Task<BlobProfile> GetProfileAsync(string email)
```

Comment:
in this situation, leave out code like new BsonDocument, and just leave off the `async` modifier instead of disabling these warnings. when the method gets implemented, then add the async modifier in.